doc/conf.py

```diff
@@ -0,0 +1,73 @@
+# Configuration file for the Sphinx documentation builder.
```

Comment:
would it be possible to move any build related files into a specific folder? to separate them from the actual markdown

Reply:
I've put all the config files into the `doc/_conf/` folder.
It works fine locally: you just need to be in that folder when running `make html` command.